lib/actual_db_schema/engine.rb

```diff
@@ -5,14 +5,16 @@ module ActualDbSchema
   class Engine < ::Rails::Engine
     isolate_namespace ActualDbSchema
 
-    initializer "actual_db_schema.append_routes", after: "add_routing_paths" do |app|
-      app.routes.append do
-        mount ActualDbSchema::Engine => "/rails"
+    initializer "actual_db_schema.append_routes", after: :add_routing_paths do |app|
+      if Rails.env.development? || Rails.env.test?
```

Comment:
WDYT if we introduce a separate config similar to "enabled" that will be responsible for loading the engine? By default, it's true when it's dev or test env. Also, what if preventing the loading engine at a higher level just not to have this "if" repeated in each block?